thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/ThirdEyeAnomalyConfiguration.java

```diff
@@ -10,8 +10,10 @@
   private boolean alert = false;
   private long id;
   private String dashboardHost;
-  private ThirdeyeSmtpConfiguration smtpConfiguration;
+  private SmtpConfiguration smtpConfiguration;
```

Comment:
that's what i wanted to call it, but we already have EmailConfiguration (mapped to our email_configuration table)